### PR TITLE
[pt] Added translations for Laravel Jetstream

### PR DESCRIPTION
**Pull request by jorgercosta**

### CODE REVIEW

File: json/pt.json [hunk 1]
```diff
@@ -1,65 +1,151 @@
...
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie o seu novo API Token. Por razões de segurança, não será mostrado novamente.",
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor insira a sua palavra-passe para confirmar que pretende terminar as suas outras sessões em browser em todos os seus dispositivos.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor forneça o endereço de e-mail da pessoa que pretende adicionar a esta equipa. O endereço de e-mail deve estar associado a uma conta que já existe.",
+    "Profile Information": "Informação de Perfil",
+    "Profile": "Perfil",
+    "Recovery Code": "Código de Recuperação",
+    "Regenerate Recovery Codes": "Regenerar Códigos de Recuperação",
     "Regards": "Atenciosamente",
     "Register": "Registar",
     "Remember Me": "Lembrar-me",
```
Comment (caouecs) [inline json/pt.json:95]: Lembrar-Me ? To be coherent with "Remember Me"

**Review by caouecs — CHANGES_REQUESTED** [review]
Obrigado but just to know for uppercase word :)

File: json/pt.json (hunk 1 above)
Comment (caouecs) [reply json/pt.json:95]: I know it's a question for all translations

**Comment by caouecs** [comment]
I merge your pull request but I think it's a good question for the coherence of "Me" and "me"

File: json/pt.json (hunk 1 above)
Comment (jorgercosta) [reply json/pt.json:95]: Not in this case: 
1. 1.  "Remember Me" and "Remember me" are both in JSON file;
2. in pt you do not use the uppercase unless, by example, related to divinity like "And then God said: Remember-Me".

And by the way i would like to thank you for the work you have done.

Cheers!

**Comment by jorgercosta** [comment]
> I merge your pull request but I think it's a good question for the coherence of "Me" and "me"

Not in this case:

"Remember Me" and "Remember me" are both in JSON file;
in pt you do not use the uppercase unless, by example, related to divinity like "And then God said: Remember-Me".
And by the way i would like to thank you for the work you have done.

Cheers!

**Pull request merged**